src/components/account/BidsPage.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react"
+
+import * as gravity from "utils/gravity"
+import { Sans, Box, Serif, Separator, BorderBox, Flex } from "@artsy/palette"
+import { RouteComponent, DebugData } from "pages/account"
+
+const DEBUG = true
+
+const last4 = (s: string) => {
+  const length = s.length
+  return s.slice(length - 5, length)
+}
+
+export const BidsPage: RouteComponent = () => {
+  const [bids, setBids] = useState<any[] | null>(null)
+  if (!bids) {
+    gravity.meBidderPositions().then(data => setBids(data))
```

Comment:
Wouldn't this be better pulled through metaphysics?

Reply:
gatsby can't build or run currently when connecting to metaphysics via `gatsby-source-graphql` - though we could still be making queries like this the old fashioned way. good point.